docs/source/import_export/main.rst

```diff
@@ -9,30 +9,29 @@ Export
 
 Use ``verdi export create`` to export a selection of nodes, codes, groups, users, or computers.
 
-* **Selection:** Nodes can be selected via identifier (PK, UUID, label) or by exporting predefined groups of nodes.
-* **Augmentation:** By default, the export function augments the selection of nodes by their parents in order to preserve the provenance.
-  For calculation nodes, the direct outputs are also added.
-  By default, comments and logs are also added for each exported node.
-  These may be excluded using the proper flags.
-  There are several other options to export specific parts of the provenance graph, starting from the selection.
-  For more information about this please read the :ref:`corresponding subsection<consistency>` of the :ref:`Provenance section<concepts_provenance>`.
+See ``verdi export create -h`` for a full list of available options.
+
+* **Selection:** Nodes can be selected via identifier (PK, UUID, label).
+  It is often convenient to first put the nodes you would like to export into a :py:class:`~aiida.orm.Group` and then to export the group using ``verdi export create -G <group-name>``.
+* **Augmentation:** By default, the export function automatically augments your selection of nodes by their logical and data provenance (and calculations by their direct outputs).
+  You can customize this behavior using command line options.
+  For details on how the provenance graph is traversed, see the section on :ref:`data consistency<consistency>`.
 * **File content:** The export file contains all information pertaining to the exported nodes: both the information stored in the database and files stored in the AiiDA repository.
-* **Compression:** By default, the export file is compressed using gzip, creating a tar ball (tar.gz).
-  It is also possible to create a compressed or uncompressed zip file.
+* **Compression:** By default, the export file is compressed using ``gzip``, creating a tar ball (``tar.gz``).
+  Use the command line options to compress using ``zip`` instead or create an uncompressed ``zip`` archive.
 
-See ``verdi export create -h`` for a full list of available options.
 
 Import
 ++++++
 
-Use ``verdi import`` to import an AiiDA export archive generated by ``verdi export create``, either a local file or an URL.
+Use ``verdi import`` to import an AiiDA export archive generated by ``verdi export create``, either from a local file or from a URL.
```

Comment:
```suggestion
Use ``verdi import`` to import an AiiDA export archive generated by ``verdi export create``, either passing a local file or a URL.
```

Reply:
It may or may not be prudent to add the notion of lists of URLs or local file paths as well (which is supported).

Reply:
will switch to plural.